### PR TITLE
Fix switch port mapping

### PR DESCRIPTION
**Pull request by sotux**
Fix switch port mapping for below devices:
tl-wr841n-v7
netgear wnr2000-v3
netgear wndr3700 v1/v2
netgear wndr3800/wndr3800ch

### CODE REVIEW

**Comment by blogic** [comment]
patch 2/3 also touches gl-ar300 but does not mention it anywhere. there is also a 4th merge commit in this branch, please remove it

**Comment by sotux** [comment]
The 4th merge commit has been removed.
Move gl-ar300 position is just for the alphabet order for this file, because I was told this file is alphabet ordered on my previous PR. But in fact it's not so strict.

File: target/linux/ar71xx/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -421,6 +415,12 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch_port_attr "switch0" 2 led 9
 		ucidef_add_switch_port_attr "switch0" 5 led 2
 		;;
+	gl-ar300|\
+	wnr2000-v3)
```
Comment (blogic) [inline target/linux/ar71xx/base-files/etc/board.d/02_network]: this reorders it to the incorrect alphabetical location

**Comment by jow-** [comment]
Merged into my staging tree with fixed alphabetical order. Thank you!

**Pull request closed without merging**